### ISSUE TITLE
Searx 500 Internal Server Error

### ISSUE THREAD

**Issue by OrangePiPC2Searx** [bashrc/freedombone#73]
I guess GitHub automatically disables accounts made with Tor/temp mail after a few hours, but that tyrant company won't stop me.

Thanks for fixing the aarch64 issue, and keep tweeting (I prefer GNU Social though) because people read it.

I'd like to run a Searx instance on a onion service, so I add the app in the menu, got a onion link from "about this system" and a password in "my passwords". but when I try to login with my username and password which is provided to me I get:

500 Internal Server Error

Don't know what to do now, should I head to the searx repo and ask there or do you know what is wrong?

**Comment by bashrc** [comment]
I recently fixed a bug with password generation where it was appending carriage returns. You could try pulling from the repo, make install and then try reinstalling searx.

**Comment by OrangePiPC2Searx** [comment]
Tried to clone stretch branch but says it can't find it, I'll wait until its in master.

what a shame though because when I first generated a onion adress I had one like this: viewmsggxxxxxxxx.onion (gone now because removed/added searx)

username should be the user of pi account right?

**Comment by bashrc** [comment]
Yes it should just be your usual username. Master has been updated. Also see https://freedombone.net/armbian.html

**Comment by OrangePiPC2Searx** [comment]
Hello again,

Had to manually build an image for my board (update from 4.11 to 4.13 kernel didn't work) hence the delay.

Is it possible to make the .onion public? because logging doesn't work again and I'm happy to share my resources to other people.

Thanks

**Comment by bashrc** [comment]
I'll add an option for that. My main concern with putting up a search engine and making it fully public is that millions of web users could then overload your small SBC with requests, or DOS it, but the existing firewall rate limits might catch that situation anyway.

**Comment by bashrc** [comment]
Made searx logins optional. https://freedombone.net/app_searx.html

**Comment by OrangePiPC2** [comment]
Thanks again,

Now I have 502 Bad Gateway, and onion address won't show up after searx in "about this system" when reinstalling.

**Comment by bashrc** [comment]
That indicates that the searx daemon isn't running for some reason.
